### DISCUSSION TITLE
Need to increase replicas for vault-configurer pods

### DISCUSSION THREAD

**Discussion by kapil7148** [bank-vaults/vault-operator#255] [Q&A] [answered]
Hi Team, 
I am using helm chart for vault-operator, after deploying helm chart , i am getting vault, vault-operator and vault-configurer pods , i need to increase replicas for vault-configurer pods how to do that ? Can someone provide me the code here , like anything need to add in crd or anything ,

**Comment by kapil7148** [comment]
@sagikazarmark , @ramizpolic could you please help here ?

**Comment by akijakya** [comment]
Hi @kapil7148, there is currently no way of my knowing to increase the replicas of the configurer via configuration, only manually by editing the already existing deployment. Although I don't really see the reason for it, I think for what it is doing, 1 pod should be enough!

**Reply by kapil7148** [reply]
Hi, basically what is use of vault-configurer pods ?

**Reply by ramizpolic** [reply, marked as answer]
The configurer ensures that Vault instance is in predefined state based on the provided configuration. The configurer params can be found at https://bank-vaults.dev/docs/external-configuration/. 
The reason we have a single Pod is to ensure that we are not issuing multiple control ops which otherwise might create additional issues with ensuring the state consistency of Vault. 
I would advise to review your current use-case and decide if increasing the configurer pods is actually necessary.

**Reply by kapil7148** [reply]
Thanks @ramizpolic @akijakya 
Just wanted to know is it ok to have 3 pods for vault, vault-secret-webhook and vault-operator and 1 pod for vault-configurer in production?

**Reply by akijakya** [reply]
I think for the `operator` and `configurer` 1-1 pod should be enough, for Vault it makes sense to have 3 (see also in the [docs](https://bank-vaults.dev/docs/operator/#ha-setup-with-raft)) and for the `webhook` as well (depends on the number of workloads you would like to inject secrets into).
